src/libsync/creds/httpcredentials.cpp

```diff
@@ -600,13 +611,15 @@ void HttpCredentials::slotWriteClientKeyPEMJobDone(QKeychain::Job *finishedJob)
 
 void HttpCredentials::slotWritePasswordToKeychain()
 {
-    auto *job = new QKeychain::WritePasswordJob(Theme::instance()->appName());
-    addSettingsToJob(_account, job);
-    job->setInsecureFallback(false);
-    connect(job, &QKeychain::Job::finished, this, &HttpCredentials::slotWriteJobDone);
-    job->setKey(keychainKey(_account->url().toString(), _user, _account->id()));
-    job->setTextData(isUsingOAuth() ? _refreshToken : _password);
-    job->start();
+    QtConcurrent::run([this] {
+        auto *job = new QKeychain::WritePasswordJob(Theme::instance()->appName());
```

Comment:
@claucambra since you are here would be nice to make all `auto *job` to `const auto job`

Reply:
Done and done